### ISSUE TITLE
[CAIA A11y Support]: Benefits and Claims, CH 31/VRE

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#61718]
### What do you need support with?

...
Maintenance audit of 28-1900 (CH31/VRE)

### When do you need this support by?

7/31/2023

### How urgent is this request?

A little urgent, we need this support before the date we provided above

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/vocational-rehab/README.md)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?sender=u0e8ac1d6d7681bb7e1b80558)
- [Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues?q=is%3Aissue+is%3Aopen+label%3Aaccessibility+label%3ACH31)
- [Form](https://www.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
...

### Who's on your team?

Team name: Non-Disability (Team Umbrella)
OCTO-DE product owner: Matt Self
Product manager: Laura Steele
Designer: Fiorella Geedi
FE engineer: Thomas Blackwell
Product/team Slack channel: benefits-ce-non_disability_exp
...

### Do you have an accessibility champion on your team?

No, we don't have a champion (yet)

**Comment by steele-lm** [comment]
@tblackwe

**Comment by sara-amanda** [comment]
## CAIA Notes
This ticket contains 1 form, with 10 open issues, and a maintenance audit is being requested before 7/31/2023. This is ticket 3 of 4 tickets. 

### 1 Form

1. 28-1900 (CH31/VRE)

### Tickets
-  **10 open issues**
- 12 closed

### :warning: Login Needed
_@steele-lm Please provide a link to where we should grab the testing login for this form._  

cc: @coforma-terry @ohgreatkate

**Comment by coforma-jamie** [comment]
### CAIA INVENTORY UPDATE

#### PDF VERSIONS (1)
- [VA Form 28-1900](https://www.vba.va.gov/pubs/forms/VBA-28-1900-ARE.pdf)
#### DIGITAL VERSIONS (1)
- [Apply for Veteran Readiness and Employment with VA Form 28-1900
](https://www.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)

**Comment by ohgreatkate** [comment]
@steele-lm Please provide a user profile to test the form

**Comment by steele-lm** [comment]
@ohgreatkate, @fiorella-io is going to work on setting up a user within the CH31/VRE form in staging.

**Comment by coforma-jamie** [comment]
@steele-lm @fiorella-io, checking in on setting up a staging user. has there been any progress? :)

**Comment by fiorella-io** [comment]
@coforma-jamie Hi Jamie, I met with Kate and mentioned I used staging user [vets.gov.user+24@gmail.com](mailto:vets.gov.user+24@gmail.com). Here's the list of [staging users](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv )to find the password. 

I used that staging user to create a Ch36 PCPG form so it should have a form in the draft section at the bottom of MyVA

I also shared a [current form flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1691434304975/3f0c405c820c184d21393b3bbd76324a5ee7c677?sender=u4e1fb0748c58370c03f33409) including screenshots of all the pages in this form

**Comment by fiorella-io** [comment]
@coforma-jamie @steele-lm I've just realized VA doesn't save the Ch36 form as a draft! 😱
<img width="952" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/91498500/8fd4296e-4117-41b4-a9ec-d203e1a87ed1">

Jamie after you sign in [go to this link](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction) and click on "Continue your application" to access the draft claim
<img width="538" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/91498500/f72205e7-5136-4fc5-89a1-737da0a4b815">

**Comment by ohgreatkate** [comment]
@fiorella-io no worries about it not saving the progress, we will want to be checking every page either way, but this ticket is for VRE CH31, from 28-1900, for now, we will use that same un&p and reach out again if we run into any issues! TY!!

**Comment by ohgreatkate** [comment]
> @coforma-jamie @steele-lm I've just realized VA doesn't save the Ch36 form as a draft! 😱 <img alt="image" width="952" src="https://user-images.githubusercontent.com/91498500/261455637-8fd4296e-4117-41b4-a9ec-d203e1a87ed1.png">
> 
> Jamie after you sign in [go to this link](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction) and click on "Continue your application" to access the draft claim <img alt="image" width="538" src="https://user-images.githubusercontent.com/91498500/261455750-f72205e7-5136-4fc5-89a1-737da0a4b815.png">

Great Catch, I am seeing this on two forms currently and have reached out to the larger CAIA team for advice and will follow up once I have insights! :)

**Comment by coforma-jamie** [comment]
@steele-lm I had a couple of questions for you about the third page of the form, [Communication Preferences](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/communication-preferences), before I write up a ticket about it. Can you answer these questions:

- If you say no to e-VA and tele-counseling, what's the alternative? In-person counseling?
- Do the meeting time preferences apply to tele-counseling, in-person counseling, or both?

**Comment by steele-lm** [comment]
@coforma-jamie Donna Robinson provided the following answers...

- Yes, the Veteran always has the option as to whether they want to opt into electronic communications through e-VA and if they want virtual counseling appointments. If they opt out, the alternative would be in-person visits and email communication.
- For the meeting preferences, in this particular scenario, the meeting is referring to the initial intake appointment (either in-person of virtual).

**Comment by ohgreatkate** [comment]
Findings shared to VFS team through team's product channel in [Slack](https://dsva.slack.com/archives/C057W3N9K2S/p1693506392603739)
